R/kfold-cv.R

```diff
@@ -27,7 +26,6 @@ crossValidation <- function(clinical_data,
     result <- list(score = mean(cvector),
                    concordances = cvector,
                    sd = sd(cvector))
-
   }
 
   else if (method == "LASSO") {
```

Comment:
Again, see my comment above about repeated CV. The `cv.glmnet` portion is used for hyperparameter tuning. You should still use `train` and `trainControl` to do the outer CV loop. 

See image below. The inner loop is where we run `cv.glmnet`. I am currently looking up to see how this works with caret and will let you know when I find something. Otherwise we can chat on Monday.

<img width="945" alt="Screenshot 2023-11-25 at 9 59 25 AM" src="https://github.com/aashen12/VoxelProc/assets/66567109/3316f791-8018-4482-b55b-5a9492ce1dde">

Reply:
The inner loop gives you the best `lambda`, and that `lambda` is used on the test set in the outer loop.